--- word2vec/mikolov/word-analogy.c	2014-12-02 16:36:50.000000000 +0100
+++ word2vec/mikolov_before/word-analogy.c	2014-09-26 19:13:10.000000000 +0200
@@ -15,7 +15,7 @@
 #include <stdio.h>
 #include <string.h>
 #include <math.h>
-#include <malloc.h>
+#include <stdlib.h>
 
 const long long max_size = 2000;         // max length of strings
 const long long N = 40;                  // number of closest words that will be shown
